tests/test_float16.py

```diff
@@ -87,6 +89,21 @@ def range_body(iter_n, cond, total):
             self.assertTrue(np.allclose(expected, actual2))
             self.assertTrue(actual2.dtype == np.float32)
 
+    def test_convert_to_float16(self):
+        # load fp32 model from data
+        model32_name = "image_classifier32.onnx"
+        working_path = os.path.abspath(os.path.dirname(__file__))
+        data_path = os.path.join(working_path, 'data')
+        model_path = os.path.join(data_path, model32_name)
+        onnx_model32 = onnxmltools.utils.load_model(model_path)
+        input_x = np.random.rand(1, 3, 32, 32).astype(np.float32)
+        output_32 = _ort_inference(onnx_model32, {'modelInput': input_x})
+        # convert to fp16 model
+        onnx_model16 = convert_float_to_float16(onnx_model32)
+        output_16 = _ort_inference(onnx_model16, {'modelInput': input_x.astype(np.float16)})
+        # compare result
```

Comment:
This comment is not necessary. Let's remove it to keep the same style as previous ones.

Reply:
removed the comment